modules/rte_mis_mail/rte_mis_mail.links.menu.yml

```diff
@@ -0,0 +1,5 @@
+rte_mis_mail.email_sms.Settings:
+  title: Email & SMS Confirmation Settings
+  parent: system.admin
```

Comment:
I don't think we should add it at `system.admin` level. Let's move this inside the configuration menu for now.

Reply:
ok